fastapi_sqla/_pytest_plugin.py

```diff
@@ -26,7 +26,7 @@ def db_url():
     Else, host used is `localhost`: postgresql://postgres@localhost/postgres
     """
     host = "postgres" if "CI" in os.environ else "localhost"
-    return f"postgresql://postgres@{host}/postgres"
+    return f"postgresql://postgres@{host}:5432/postgres"
```

Comment:
Rather than requiring the db_url to always have a port (which I'm assuming is why you added this), could we instead default to 5432 [here](https://github.com/dialoguemd/fastapi-sqla/pull/43/files#diff-b2ab2a53b6ed37e0cc84a6d09670f9e1fb169ba79fb25898d06ab9ce56776415R17)?

Reply:
I quickly checked and we always pass the port in the sql_alchemy variable. What if we need to override? I suggest we get the port if it is set, otherwise we take 5432. What do you think?

Reply:
Yes that sounds great

Reply:
Done